test/fixtures/schema.sql

```diff
@@ -2242,4 +2242,28 @@ A test for partitioned tables$$;
         foreign key (id_a, name_a) references test.partitioned_a (id, name)
       );
     end if;
-end$do$;
\ No newline at end of file
+end$do$;
+
+create or replace function test.unnamed_json_param(json) returns json as $$
+  select $1;
+$$ language sql;
+
+create or replace function test.unnamed_text_param(text) returns text as $$
+  select $1;
+$$ language sql ;
+
+create or replace function test.unnamed_bytea_param(bytea) returns bytea as $$
+  select $1::bytea;
+$$ language sql ;
+
+create or replace function test.unnamed_int_param(int) returns int as $$
+  select $1;
+$$ language sql;
+
+create or replace function test.overloaded_unnamed_param(json) returns int as $$
+  select $1;
+$$ language sql;
+
+create or replace function test.overloaded_unnamed_param(x int, y int) returns int as $$
+  select x + y;
+$$ language sql;
```

Comment:
Having a func with an unnamed json parameter plus any number of overloaded pairs

Reply:
Somehow it doesn't show where, but the above function is:

```sql
create or replace function test.overloaded_unnamed_param(json) returns int as $$
  select $1;
$$ language sql;
```